base/stats.go

```diff
@@ -272,7 +272,7 @@ type CBLReplicationPullStats struct {
 	AttachmentPullCount *SgwIntStat `json:"attachment_pull_count"`
 	// The high watermark for the number of documents buffered during feed processing, waiting on a missing earlier sequence.
 	MaxPending *SgwIntStat `json:"max_pending"`
-	/// The total number of active replications. This metric only counts continuous pull replications.
+	// The total number of active replications. This metric only counts continuous pull replications.
```

Comment:
does this comment actually belong on the line below? outside scope of review though

Reply:
I don't think so - the stat is just not accounting for all of the cases it should be doing. The below stat is more specific (and correct)